apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisClientConstructorInterceptor.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+
+package org.apache.skywalking.apm.plugin.redisson.v3;
+
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.EnhancedInstance;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.InstanceConstructorInterceptor;
+
+/**
+ * @author zhaoyuguang
+ */
+public class RedisClientConstructorInterceptor implements InstanceConstructorInterceptor {
```

Comment:
It seems that there is no logic in this interceptor. Is this interceptor necessary, if not, delete it.

Reply:
RedisClient is the link between RedisConnection and ConnectionManager. to enhance RedisClient for bring peer(the cluster configuration information) in ConnectionManager to RedisConnection.

Reply:
Oh. You need this to add `dynamicField`, right? I recommend you to put the comments in this class, in order to tell the reader why you need an empty interceptor.

Reply:
Yes, the RedisConnection is the lowest level of redisson and netty communication class, but RedisConnection and RedisClient do not store redisson for redis cluster configuration information.

Reply:
OK，got it.